<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Mass Spectrometry Data on ExperimentHub â€¢ MsDataHub</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Mass Spectrometry Data on ExperimentHub">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">MsDataHub</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.7.4</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/MsDataHub.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/RforMassSpectrometry/MsDataHub/" class="external-link">
    <span class="fab fa-github fa-lg"></span>

  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><link href="MsDataHub_files/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">
<script src="MsDataHub_files/htmlwidgets-1.6.4/htmlwidgets.js"></script><link href="MsDataHub_files/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="MsDataHub_files/datatables-binding-0.33/datatables.js"></script><link href="MsDataHub_files/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">
<link href="MsDataHub_files/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">
<script src="MsDataHub_files/dt-core-1.13.6/js/jquery.dataTables.min.js"></script><link href="MsDataHub_files/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">
<script src="MsDataHub_files/crosstalk-1.2.1/js/crosstalk.min.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Mass Spectrometry Data on ExperimentHub</h1>
                        <h4 data-toc-skip class="author">Laurent
Gatto</h4>
                        
      
      <small class="dont-index">Source: <a href="https://github.com/RforMassSpectrometry/MsDataHub/blob/main/vignettes/MsDataHub.Rmd" class="external-link"><code>vignettes/MsDataHub.Rmd</code></a></small>
      <div class="hidden name"><code>MsDataHub.Rmd</code></div>

    </div>

    
    
<script type="text/javascript">
document.addEventListener("DOMContentLoaded", function() {
  document.querySelector("h1").className = "title";
});
</script><script type="text/javascript">
document.addEventListener("DOMContentLoaded", function() {
  var links = document.links;  
  for (var i = 0, linksLength = links.length; i < linksLength; i++)
    if (links[i].hostname != window.location.hostname)
      links[i].target = '_blank';
});
</script><div class="section level2">
<h2 id="introduction">Introduction<a class="anchor" aria-label="anchor" href="#introduction"></a>
</h2>
<p>The <code>MsDataHub</code> package provides example mass spectrometry
data, peptide spectrum matches or quantitative data from proteomics and
metabolomics experiments. The data are served through the
<code>ExperimentHub</code> infrastructure, which allows download them
only ones and cache them for further use. Currently available data are
summarised in the table below and details in the next section.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="st"><a href="https://rformassspectrometry.github.io/MsDataHub" class="external-link">"MsDataHub"</a></span><span class="op">)</span></span>
<span><span class="fu">DT</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/DT/man/datatable.html" class="external-link">datatable</a></span><span class="op">(</span><span class="fu"><a href="../reference/MsDataHub.html">MsDataHub</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span></span></code></pre></div>
<div class="datatables html-widget html-fill-item" id="htmlwidget-ac96cb3ee4656e2e9ec3" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-ac96cb3ee4656e2e9ec3">{"x":{"filter":"none","vertical":false,"data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17"],["ko15.CDF","cptac_a_b_c_peptides.txt","cptac_a_b_peptides.txt","cptac_peptides.txt","TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01.20141210.mzid","TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01.20141210.mzML.gz","X20171016_POOL_POS_1_105.134.mzML","X20171016_POOL_POS_3_105.134.mzML","PestMix1_DDA.mzML","PestMix1_SWATH.mzML","benchmarkingDIA.tsv","Report.Derks2022.plexDIA.tsv","Ai2025_aCMs_report.tsv","Ai2025_iCMs_report.tsv","crap_gpm.fasta","crap_ccp.fasta","crap_maxquant.fasta.gz"],["Raw metabolomics MS file in netCDF format. See ?ko15.CDF for details.","Conditions A, B and C of the CPTAC quantitative proteomics data (tab-delimited format). See ?cptac for details.","Conditions A and B of the CPTAC quantitative proteomics data (tab-delimited format). See ?cptac for details.","CPTAC quantitative proteomics data (tab-delimited format). See ?cptac for details.","Peptide spectrum matches from the PDX000001 experiment. See ?PXD000001 for details.","Raw MS data from the PDX000001 experiment, in mzML format. See ?PXD000001 for details.","AB Sciex LC-MS data file (injection index 1), in mzML format. See ?sciex for details.","AB Sciex LC-MS data file (injection index 19), in mzML format. See ?sciex for details.","Triple TOF DDA raw data, in mzML format. See ?TripleTOF for details.","Triple TOF SWATH raw data, in mzML format. See ?TripleTOF for details.","Output of DIA-NN software (report.tsv)","Derk et al. (2022) single-cell proteomics plexDIA results (DIA-NN report.tsv).","Ai et al. (2025) Single Cell Proteomics Reveals Specific Cellular Subtypes in Cardiomyocytes Derived from Human iPSCs and Adult Hearts.","Ai et al. (2025) Single Cell Proteomics Reveals Specific Cellular Subtypes in Cardiomyocytes Derived from Human iPSCs and Adult Hearts.","common Repository of Adventitious Proteins (cRAP) from the Global Proteome Machine (GPM)","common Repository of Adventitious Proteins (cRAP) from the Cambridge Centre for Proteomics (CCP)","MaxQuant's contaminant database"],[3.17,3.17,3.17,3.17,3.17,3.17,3.17,3.17,3.17,3.17,3.17,3.19,3.21,3.21,3.21,3.21,3.21],[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],["CDF","TXT","TXT","TXT","mzid","mzML","mzML","mzML","mzML","mzML","TSV","TSV","TSV","TSV","FASTA","FASTA","FASTA"],["https://bioconductor.org/packages/3.16/data/experiment/html/msdata.html","https://uclouvain-cbio.github.io/WSBIM2122/data/cptac_a_b_c_peptides.txt","https://bioconductor.org/packages/3.16/data/experiment/html/msdata.html","https://raw.githubusercontent.com/statOmics/PDA/data/quantification/fullCptacDatasSetNotForTutorial/peptides.txt","https://bioconductor.org/packages/3.16/data/experiment/html/msdata.html","https://ftp.pride.ebi.ac.uk/pride/data/archive/2012/03/PXD000001/","https://bioconductor.org/packages/3.16/data/experiment/html/msdata.html","https://bioconductor.org/packages/3.16/data/experiment/html/msdata.html","https://bioconductor.org/packages/3.16/data/experiment/html/msdata.html","https://bioconductor.org/packages/3.16/data/experiment/html/msdata.html","https://zenodo.org/records/8063173","https://drive.google.com/drive/folders/1pUC2zgXKtKYn22mlor0lmUDK0frgwL_-","ftp://massive.ucsd.edu/v07/MSV000094438/updates/2024-10-23_bineka_5deafa61/other/Supplementary%20Files/aCMs/report.tsv","ftp://massive.ucsd.edu/v07/MSV000094438/updates/2024-10-23_bineka_5deafa61/other/Supplementary%20Files/iCMs/report1.tsv","https://zenodo.org/records/15115102/files/crap_gpm.fasta","https://zenodo.org/records/15115102/files/crap_ccp.fasta","https://zenodo.org/records/15115102/files/crap_maxquant.fasta.gz"],[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1],["Mus musculus","Saccharomyces cerevisiae","Saccharomyces cerevisiae","Saccharomyces cerevisiae","Erwinia carotovora","Erwinia carotovora","Homo sapiens","Homo sapiens","","","Homo sapiens","Homo sapiens","Homo sapiens","Homo sapiens","","",""],[10090,4932,4932,4932,554,554,9606,9606,null,null,9606,9606,9606,9606,null,null,null],[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null],["Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;","Laurent Gatto &lt;laurent.gatto@uclouvain.be&gt;"],["Spectra","data.frame","data.frame","data.frame","Spectra","PSM","Spectra","Spectra","Spectra","Spectra","data.frame","data.frame","data.frame","data.frame","AAStringSet","AAStringSet","AAStringSet"],["FilePath","FilePath","FilePath","FilePath","FilePath","FilePath","FilePath","FilePath","FilePath","FilePath","FilePath","FilePath","FilePath","FilePath","FilePath","FilePath","FilePath"],["","","","","","","","","","","https://zenodo.org/","https://zenodo.org/","ftp://massive.ucsd.edu/","ftp://massive.ucsd.edu/","https://zenodo.org/","https://zenodo.org/","https://zenodo.org/"],["MsDataHub/cdf/ko15.CDF","MsDataHub/cptac/cptac_a_b_c_peptides.txt","MsDataHub/cptac/cptac_a_b_peptides.txt","MsDataHub/cptac/cptac_peptides.txt","MsDataHub/PXD000001/TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01-20141210.mzid","MsDataHub/PXD000001/TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01-20141210.mzML.gz","MsDataHub/sciex/20171016_POOL_POS_1_105-134.mzML","MsDataHub/sciex/20171016_POOL_POS_3_105-134.mzML","MsDataHub/TripleTOF-SWATH/PestMix1_DDA.mzML","MsDataHub/TripleTOF-SWATH/PestMix1_SWATH.mzML","record/8063173/files/benchmarkingDIA.tsv","records/10938597/files/Report-Derks2022-plexDIA.tsv","v07/MSV000094438/updates/2024-10-23_bineka_5deafa61/other/Supplementary%20Files/aCMs/report.tsv","v07/MSV000094438/updates/2024-10-23_bineka_5deafa61/other/Supplementary%20Files/iCMs/report1.tsv","records/15115102/files/crap_gpm.fasta","records/15115102/files/crap_ccp.fasta","records/15115102/files/crap_maxquant.fasta.gz"],[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Title<\/th>\n      <th>Description<\/th>\n      <th>BiocVersion<\/th>\n      <th>Genome<\/th>\n      <th>SourceType<\/th>\n      <th>SourceUrl<\/th>\n      <th>SourceVersion<\/th>\n      <th>Species<\/th>\n      <th>TaxonomyId<\/th>\n      <th>Coordinate_1_based<\/th>\n      <th>DataProvider<\/th>\n      <th>Maintainer<\/th>\n      <th>RDataClass<\/th>\n      <th>DispatchClass<\/th>\n      <th>Location_Prefix<\/th>\n      <th>RDataPath<\/th>\n      <th>Tags<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[3,7,9]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Title","targets":1},{"name":"Description","targets":2},{"name":"BiocVersion","targets":3},{"name":"Genome","targets":4},{"name":"SourceType","targets":5},{"name":"SourceUrl","targets":6},{"name":"SourceVersion","targets":7},{"name":"Species","targets":8},{"name":"TaxonomyId","targets":9},{"name":"Coordinate_1_based","targets":10},{"name":"DataProvider","targets":11},{"name":"Maintainer","targets":12},{"name":"RDataClass","targets":13},{"name":"DispatchClass","targets":14},{"name":"Location_Prefix","targets":15},{"name":"RDataPath","targets":16},{"name":"Tags","targets":17}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level2">
<h2 id="installation">Installation<a class="anchor" aria-label="anchor" href="#installation"></a>
</h2>
<p>To install the package:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">require</a></span><span class="op">(</span><span class="st"><a href="https://bioconductor.github.io/BiocManager/" class="external-link">"BiocManager"</a></span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html" class="external-link">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span></span>
<span></span>
<span><span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://bioconductor.github.io/BiocManager/reference/install.html" class="external-link">install</a></span><span class="op">(</span><span class="st">"MsDataHub"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="available-data">Available data<a class="anchor" aria-label="anchor" href="#available-data"></a>
</h2>
<div class="section level3">
<h3 id="tripletof">TripleTOF<a class="anchor" aria-label="anchor" href="#tripletof"></a>
</h3>
<ul>
<li>Type: Raw MS data</li>
<li>Files: <code>PestMix1_DDA.mzML</code> and
<code>PestMix1_SWATH.mzML</code>
</li>
<li>More details: <code><a href="../reference/TripleTOF.html">?TripleTOF</a></code>
</li>
</ul>
<p>Load with</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">f</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/TripleTOF.html">PestMix1_DDA.mzML</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## see ?MsDataHub and browseVignettes('MsDataHub') for documentation</span></span></code></pre>
<pre><code><span><span class="co">## loading from cache</span></span></code></pre>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/RforMassSpectrometry/Spectra" class="external-link">Spectra</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/Spectra/man/Spectra.html" class="external-link">Spectra</a></span><span class="op">(</span><span class="va">f</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## MSn data (Spectra) with 7602 spectra in a MsBackendMzR backend:</span></span>
<span><span class="co">##        msLevel     rtime scanIndex</span></span>
<span><span class="co">##      &lt;integer&gt; &lt;numeric&gt; &lt;integer&gt;</span></span>
<span><span class="co">## 1            1     0.231         1</span></span>
<span><span class="co">## 2            1     0.351         2</span></span>
<span><span class="co">## 3            1     0.471         3</span></span>
<span><span class="co">## 4            1     0.591         4</span></span>
<span><span class="co">## 5            1     0.711         5</span></span>
<span><span class="co">## ...        ...       ...       ...</span></span>
<span><span class="co">## 7598         1   899.491      7598</span></span>
<span><span class="co">## 7599         1   899.613      7599</span></span>
<span><span class="co">## 7600         1   899.747      7600</span></span>
<span><span class="co">## 7601         1   899.872      7601</span></span>
<span><span class="co">## 7602         1   899.993      7602</span></span>
<span><span class="co">##  ... 34 more variables/columns.</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## file(s):</span></span>
<span><span class="co">## 820a2ede30c0_7861</span></span></code></pre>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">f</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/TripleTOF.html">PestMix1_SWATH.mzML</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## see ?MsDataHub and browseVignettes('MsDataHub') for documentation</span></span></code></pre>
<pre><code><span><span class="co">## loading from cache</span></span></code></pre>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/Spectra/man/Spectra.html" class="external-link">Spectra</a></span><span class="op">(</span><span class="va">f</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## MSn data (Spectra) with 8999 spectra in a MsBackendMzR backend:</span></span>
<span><span class="co">##        msLevel     rtime scanIndex</span></span>
<span><span class="co">##      &lt;integer&gt; &lt;numeric&gt; &lt;integer&gt;</span></span>
<span><span class="co">## 1            2     0.203         1</span></span>
<span><span class="co">## 2            2     0.300         2</span></span>
<span><span class="co">## 3            2     0.397         3</span></span>
<span><span class="co">## 4            2     0.494         4</span></span>
<span><span class="co">## 5            2     0.591         5</span></span>
<span><span class="co">## ...        ...       ...       ...</span></span>
<span><span class="co">## 8995         2   899.527      8995</span></span>
<span><span class="co">## 8996         2   899.624      8996</span></span>
<span><span class="co">## 8997         2   899.721      8997</span></span>
<span><span class="co">## 8998         2   899.818      8998</span></span>
<span><span class="co">## 8999         2   899.915      8999</span></span>
<span><span class="co">##  ... 34 more variables/columns.</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## file(s):</span></span>
<span><span class="co">## 820a11a9b46b_7862</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="sciex">sciex<a class="anchor" aria-label="anchor" href="#sciex"></a>
</h3>
<ul>
<li>Type: Raw MS data</li>
<li>Files: <code>20171016_POOL_POS_1_105-134.mzML</code> and
<code>20171016_POOL_POS_3_105-134.mzML</code>
</li>
<li>More details: <code><a href="../reference/sciex.html">?sciex</a></code>
</li>
</ul>
<p>Load with</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">f</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/sciex.html">X20171016_POOL_POS_1_105.134.mzML</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## see ?MsDataHub and browseVignettes('MsDataHub') for documentation</span></span></code></pre>
<pre><code><span><span class="co">## loading from cache</span></span></code></pre>
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/Spectra/man/Spectra.html" class="external-link">Spectra</a></span><span class="op">(</span><span class="va">f</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## MSn data (Spectra) with 931 spectra in a MsBackendMzR backend:</span></span>
<span><span class="co">##       msLevel     rtime scanIndex</span></span>
<span><span class="co">##     &lt;integer&gt; &lt;numeric&gt; &lt;integer&gt;</span></span>
<span><span class="co">## 1           1     0.280         1</span></span>
<span><span class="co">## 2           1     0.559         2</span></span>
<span><span class="co">## 3           1     0.838         3</span></span>
<span><span class="co">## 4           1     1.117         4</span></span>
<span><span class="co">## 5           1     1.396         5</span></span>
<span><span class="co">## ...       ...       ...       ...</span></span>
<span><span class="co">## 927         1   258.641       927</span></span>
<span><span class="co">## 928         1   258.920       928</span></span>
<span><span class="co">## 929         1   259.199       929</span></span>
<span><span class="co">## 930         1   259.478       930</span></span>
<span><span class="co">## 931         1   259.757       931</span></span>
<span><span class="co">##  ... 34 more variables/columns.</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## file(s):</span></span>
<span><span class="co">## 820ad827835_7859</span></span></code></pre>
<div class="sourceCode" id="cb18"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">f</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/sciex.html">X20171016_POOL_POS_3_105.134.mzML</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## see ?MsDataHub and browseVignettes('MsDataHub') for documentation</span></span></code></pre>
<pre><code><span><span class="co">## loading from cache</span></span></code></pre>
<div class="sourceCode" id="cb21"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/Spectra/man/Spectra.html" class="external-link">Spectra</a></span><span class="op">(</span><span class="va">f</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## MSn data (Spectra) with 931 spectra in a MsBackendMzR backend:</span></span>
<span><span class="co">##       msLevel     rtime scanIndex</span></span>
<span><span class="co">##     &lt;integer&gt; &lt;numeric&gt; &lt;integer&gt;</span></span>
<span><span class="co">## 1           1     0.275         1</span></span>
<span><span class="co">## 2           1     0.554         2</span></span>
<span><span class="co">## 3           1     0.833         3</span></span>
<span><span class="co">## 4           1     1.112         4</span></span>
<span><span class="co">## 5           1     1.391         5</span></span>
<span><span class="co">## ...       ...       ...       ...</span></span>
<span><span class="co">## 927         1   258.636       927</span></span>
<span><span class="co">## 928         1   258.915       928</span></span>
<span><span class="co">## 929         1   259.194       929</span></span>
<span><span class="co">## 930         1   259.473       930</span></span>
<span><span class="co">## 931         1   259.752       931</span></span>
<span><span class="co">##  ... 34 more variables/columns.</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## file(s):</span></span>
<span><span class="co">## 820a2780d9b2_7860</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="pxd000001">PXD000001<a class="anchor" aria-label="anchor" href="#pxd000001"></a>
</h3>
<ul>
<li>Type: Raw MS data and peptide spectrum matches</li>
<li>Files:
<code>TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01-20141210.mzML.gz</code>
and
<code>TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01-20141210.mzid</code>
</li>
<li>More details: <code>?PDX000001</code>
</li>
</ul>
<p>Load with</p>
<div class="sourceCode" id="cb23"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">f</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/PXD000001.html">TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01.20141210.mzML.gz</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## see ?MsDataHub and browseVignettes('MsDataHub') for documentation</span></span></code></pre>
<pre><code><span><span class="co">## loading from cache</span></span></code></pre>
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/Spectra/man/Spectra.html" class="external-link">Spectra</a></span><span class="op">(</span><span class="va">f</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## MSn data (Spectra) with 7534 spectra in a MsBackendMzR backend:</span></span>
<span><span class="co">##        msLevel     rtime scanIndex</span></span>
<span><span class="co">##      &lt;integer&gt; &lt;numeric&gt; &lt;integer&gt;</span></span>
<span><span class="co">## 1            1    0.4584         1</span></span>
<span><span class="co">## 2            1    0.9725         2</span></span>
<span><span class="co">## 3            1    1.8524         3</span></span>
<span><span class="co">## 4            1    2.7424         4</span></span>
<span><span class="co">## 5            1    3.6124         5</span></span>
<span><span class="co">## ...        ...       ...       ...</span></span>
<span><span class="co">## 7530         2   3600.47      7530</span></span>
<span><span class="co">## 7531         2   3600.83      7531</span></span>
<span><span class="co">## 7532         2   3601.18      7532</span></span>
<span><span class="co">## 7533         2   3601.57      7533</span></span>
<span><span class="co">## 7534         2   3601.98      7534</span></span>
<span><span class="co">##  ... 34 more variables/columns.</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## file(s):</span></span>
<span><span class="co">## 820a6b11fdf4_7858</span></span></code></pre>
<div class="sourceCode" id="cb28"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">f</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/PXD000001.html">TMT_Erwinia_1uLSike_Top10HCD_isol2_45stepped_60min_01.20141210.mzid</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## see ?MsDataHub and browseVignettes('MsDataHub') for documentation</span></span></code></pre>
<pre><code><span><span class="co">## loading from cache</span></span></code></pre>
<div class="sourceCode" id="cb31"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/RforMassSpectrometry/PSM" class="external-link">PSMatch</a></span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/PSMatch/man/PSM.html" class="external-link">PSM</a></span><span class="op">(</span><span class="va">f</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## PSM with 5802 rows and 35 columns.</span></span>
<span><span class="co">## names(35): sequence spectrumID ... subReplacementResidue subLocation</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="cptac">CPTAC<a class="anchor" aria-label="anchor" href="#cptac"></a>
</h3>
<ul>
<li>Type: tab-delimited quantitative proteomics data tables (as produced
by MaxQuant)</li>
<li>Files: <code>cptac_a_b_c_peptides.txt</code>,
<code>cptac_a_b_peptides.txt</code> and
<code>cptac_peptides.txt</code>
</li>
<li>More details: <code><a href="../reference/cptac.html">?cptac</a></code>
</li>
</ul>
<p>Load with</p>
<div class="sourceCode" id="cb33"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/RforMassSpectrometry/QFeatures" class="external-link">QFeatures</a></span><span class="op">)</span></span>
<span><span class="va">f</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/cptac.html">cptac_peptides.txt</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## see ?MsDataHub and browseVignettes('MsDataHub') for documentation</span></span></code></pre>
<pre><code><span><span class="co">## loading from cache</span></span></code></pre>
<div class="sourceCode" id="cb36"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">ecols</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/grep.html" class="external-link">grep</a></span><span class="op">(</span><span class="st">"Intensity\\."</span>, <span class="fu"><a href="https://rdrr.io/r/base/names.html" class="external-link">names</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/utils/read.table.html" class="external-link">read.delim</a></span><span class="op">(</span><span class="va">f</span><span class="op">)</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/QFeatures/man/readQFeatures.html" class="external-link">readSummarizedExperiment</a></span><span class="op">(</span><span class="va">f</span>, <span class="va">ecols</span>, sep <span class="op">=</span> <span class="st">"\t"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## class: SummarizedExperiment </span></span>
<span><span class="co">## dim: 11466 45 </span></span>
<span><span class="co">## metadata(0):</span></span>
<span><span class="co">## assays(1): ''</span></span>
<span><span class="co">## rownames(11466): 1 2 ... 11465 11466</span></span>
<span><span class="co">## rowData names(143): Sequence N.term.cleavage.window ...</span></span>
<span><span class="co">##   Oxidation..M..site.IDs MS.MS.Count</span></span>
<span><span class="co">## colnames(45): Intensity.6A_1 Intensity.6A_2 ... Intensity.6E_8</span></span>
<span><span class="co">##   Intensity.6E_9</span></span>
<span><span class="co">## colData names(0):</span></span></code></pre>
<div class="sourceCode" id="cb38"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/cptac.html">cptac_a_b_c_peptides.txt</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## see ?MsDataHub and browseVignettes('MsDataHub') for documentation</span></span></code></pre>
<pre><code><span><span class="co">## loading from cache</span></span></code></pre>
<pre><code><span><span class="co">##                                                  EH7804 </span></span>
<span><span class="co">## "/github/home/.cache/R/ExperimentHub/820a31199024_7854"</span></span></code></pre>
<div class="sourceCode" id="cb42"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="../reference/cptac.html">cptac_a_b_peptides.txt</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## see ?MsDataHub and browseVignettes('MsDataHub') for documentation</span></span>
<span><span class="co">## loading from cache</span></span></code></pre>
<pre><code><span><span class="co">##                                                  EH7805 </span></span>
<span><span class="co">## "/github/home/.cache/R/ExperimentHub/820a47e9fa58_7855"</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="faah-ko">FAAH KO<a class="anchor" aria-label="anchor" href="#faah-ko"></a>
</h3>
<ul>
<li>Type: Raw MS data, in netCDF format.</li>
<li>File: <code>ko15.CDF</code>
</li>
<li>More details: <code><a href="../reference/cdf.html">?cdf</a></code>
</li>
</ul>
<p>Load with</p>
<div class="sourceCode" id="cb45"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">f</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/cdf.html">ko15.CDF</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## see ?MsDataHub and browseVignettes('MsDataHub') for documentation</span></span></code></pre>
<pre><code><span><span class="co">## loading from cache</span></span></code></pre>
<div class="sourceCode" id="cb48"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/Spectra/man/Spectra.html" class="external-link">Spectra</a></span><span class="op">(</span><span class="va">f</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## MSn data (Spectra) with 1278 spectra in a MsBackendMzR backend:</span></span>
<span><span class="co">##        msLevel     rtime scanIndex</span></span>
<span><span class="co">##      &lt;integer&gt; &lt;numeric&gt; &lt;integer&gt;</span></span>
<span><span class="co">## 1            1   2501.38         1</span></span>
<span><span class="co">## 2            1   2502.94         2</span></span>
<span><span class="co">## 3            1   2504.51         3</span></span>
<span><span class="co">## 4            1   2506.07         4</span></span>
<span><span class="co">## 5            1   2507.64         5</span></span>
<span><span class="co">## ...        ...       ...       ...</span></span>
<span><span class="co">## 1274         1   4493.56      1274</span></span>
<span><span class="co">## 1275         1   4495.13      1275</span></span>
<span><span class="co">## 1276         1   4496.69      1276</span></span>
<span><span class="co">## 1277         1   4498.26      1277</span></span>
<span><span class="co">## 1278         1   4499.82      1278</span></span>
<span><span class="co">##  ... 33 more variables/columns.</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## file(s):</span></span>
<span><span class="co">## 820a755eeac5_7853</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="dia-nn-software-outputs">DIA-NN software outputs<a class="anchor" aria-label="anchor" href="#dia-nn-software-outputs"></a>
</h3>
<ul>
<li>Type: tab-delimited DIA quantitative proteomics data tables produced
by <a href="https://github.com/vdemichev/DiaNN" class="external-link">DIA-NN</a>.</li>
<li>Files:
<ul>
<li>Label-free DIA: <code>benchmarkingDIA.tsv</code>
</li>
<li>mTRAQ plexDIA: <code>Report.Derks2022.plexDIA.tsv</code>
</li>
</ul>
</li>
<li>More details: <code><a href="../reference/benchmarkingDIA.html">?benchmarkingDIA.tsv</a></code> and
<code><a href="../reference/Report.Derks2022.plexDIA.html">?Report.Derks2022.plexDIA.tsv</a></code>
</li>
</ul>
<p>Load with</p>
<div class="sourceCode" id="cb50"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/RforMassSpectrometry/QFeatures" class="external-link">QFeatures</a></span><span class="op">)</span></span>
<span><span class="va">lfdia</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/utils/read.table.html" class="external-link">read.delim</a></span><span class="op">(</span><span class="fu">MsDataHub</span><span class="fu">::</span><span class="fu"><a href="../reference/benchmarkingDIA.html">benchmarkingDIA.tsv</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/QFeatures/man/readQFeaturesFromDIANN.html" class="external-link">readQFeaturesFromDIANN</a></span><span class="op">(</span><span class="va">lfdia</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## An instance of class QFeatures containing 24 set(s):</span></span>
<span><span class="co">##  [1] U:\712006-Proteomics\Issues\Issue 253\DIANN\raw-data\RD139_Overlap_UPS1_0_1fmol_inj1.mzML: SummarizedExperiment with 28980 rows and 1 columns </span></span>
<span><span class="co">##  [2] U:\712006-Proteomics\Issues\Issue 253\DIANN\raw-data\RD139_Overlap_UPS1_0_1fmol_inj2.mzML: SummarizedExperiment with 29495 rows and 1 columns </span></span>
<span><span class="co">##  [3] U:\712006-Proteomics\Issues\Issue 253\DIANN\raw-data\RD139_Overlap_UPS1_0_1fmol_inj3.mzML: SummarizedExperiment with 29210 rows and 1 columns </span></span>
<span><span class="co">##  ...</span></span>
<span><span class="co">##  [22] U:\712006-Proteomics\Issues\Issue 253\DIANN\raw-data\RD139_Overlap_UPS1_5fmol_inj1.mzML: SummarizedExperiment with 30941 rows and 1 columns </span></span>
<span><span class="co">##  [23] U:\712006-Proteomics\Issues\Issue 253\DIANN\raw-data\RD139_Overlap_UPS1_5fmol_inj2.mzML: SummarizedExperiment with 30321 rows and 1 columns </span></span>
<span><span class="co">##  [24] U:\712006-Proteomics\Issues\Issue 253\DIANN\raw-data\RD139_Overlap_UPS1_5fmol_inj3.mzML: SummarizedExperiment with 24168 rows and 1 columns</span></span></code></pre>
<div class="sourceCode" id="cb52"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">plexdia</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/utils/read.table.html" class="external-link">read.delim</a></span><span class="op">(</span><span class="fu">MsDataHub</span><span class="fu">::</span><span class="fu"><a href="../reference/Report.Derks2022.plexDIA.html">Report.Derks2022.plexDIA.tsv</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/QFeatures/man/readQFeaturesFromDIANN.html" class="external-link">readQFeaturesFromDIANN</a></span><span class="op">(</span><span class="va">plexdia</span>, multiplexing <span class="op">=</span> <span class="st">"mTRAQ"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## An instance of class QFeatures containing 54 set(s):</span></span>
<span><span class="co">##  [1] F:\JD\plexDIA\nPOP\wJD1146.raw: SummarizedExperiment with 2635 rows and 3 columns </span></span>
<span><span class="co">##  [2] F:\JD\plexDIA\nPOP\wJD1147.raw: SummarizedExperiment with 3000 rows and 3 columns </span></span>
<span><span class="co">##  [3] F:\JD\plexDIA\nPOP\wJD1148.raw: SummarizedExperiment with 2676 rows and 3 columns </span></span>
<span><span class="co">##  ...</span></span>
<span><span class="co">##  [52] F:\JD\plexDIA\nPOP\wJD1203.raw: SummarizedExperiment with 4441 rows and 3 columns </span></span>
<span><span class="co">##  [53] F:\JD\plexDIA\nPOP\wJD1204.raw: SummarizedExperiment with 4416 rows and 3 columns </span></span>
<span><span class="co">##  [54] F:\JD\plexDIA\nPOP\wJD1205.raw: SummarizedExperiment with 4492 rows and 3 columns</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="dia-nn-single-cell-proteomics-reports">DIA-NN single-cell proteomics reports<a class="anchor" aria-label="anchor" href="#dia-nn-single-cell-proteomics-reports"></a>
</h3>
<ul>
<li>Type: tab-delimited DIA quantitative proteomics data tables produced
by <a href="https://github.com/vdemichev/DiaNN" class="external-link">DIA-NN</a>.</li>
<li>Files:
<ul>
<li>Single-cell abel-free: <code>Ai2025_aCMs_report.tsv</code>
</li>
<li>Single-cell label-free: <code>Ai2025_iCMs_report.tsv</code>
</li>
</ul>
</li>
<li>More details: <code><a href="../reference/Ai2025.html">?Ai2025</a></code>.</li>
</ul>
</div>
<div class="section level3">
<h3 id="proteomics-contaminant-databases">Proteomics contaminant databases<a class="anchor" aria-label="anchor" href="#proteomics-contaminant-databases"></a>
</h3>
<ul>
<li>Type: fasta files, as documented in <code>camprotR</code>â€™s <a href="https://cambridgecentreforproteomics.github.io/camprotR/articles/crap.html" class="external-link">cRAP
databases</a> vignette.</li>
<li>Files:
<ul>
<li>
<code>crap_gpm.fasta</code>: the common Repository of Adventitious
Proteins (cRAP) from the Global Proteome Machine (GPM)
organisation.</li>
<li>
<code>crap_ccp.fasta</code>: Cambridge Centre for Proteomicsâ€™ own
cRAP fasta database.</li>
<li>
<code>crap_maxquant.fasta.gz</code>: MaxQuantâ€™s contaminant
database.</li>
</ul>
</li>
<li>More details: <code><a href="../reference/cRAP.html">?cRAP</a></code>.</li>
</ul>
</div>
</div>
<div class="section level2">
<h2 id="adding-data-to-msdatahub">Adding data to <code>MsDataHub</code><a class="anchor" aria-label="anchor" href="#adding-data-to-msdatahub"></a>
</h2>
<ol style="list-style-type: decimal">
<li>If you would like additional dataset to <code>MsDataHub</code>,
start by opening an <a href="https://github.com/rformassspectrometry/MsDataHub/issues" class="external-link">issue</a>
in the packageâ€™s GitHub repository and describe the new data. In
particular, provide information about itâ€™s provenance, its use, its
format(s) and acknowledge that the data may be shared freely with the
community without any restrictions. You may provide an open licence
specifying the terms it can be re-used, typically a CC-BY-SA
license.</li>
<li>By contribution to the package, you acknowledge that you will comply
to the R for Mass Spectrometry project <a href="https://rformassspectrometry.github.io/RforMassSpectrometry/articles/RforMassSpectrometry.html#code-of-conduct" class="external-link">code
of conduct</a>.</li>
<li>A maintainer of the package will reply to your issue, confirming
that the data can be added.</li>
<li>At this point, if you are familiar with the development of
<code>ExperimentHub</code> packages and GitHub <em>pull requests</em>,
you may directly send one that adds your data to the package. Make sure
(1) add appropriate references in the manual page and (2) to add
yourself as a contributor of the package in the DESCRIPTION file.</li>
<li>Alternatively, a maintainer will add the dataset to the package and
may require your input to make sure the documentation file is
complete.</li>
</ol>
</div>
<div class="section level2">
<h2 class="unnumbered" id="session-information">Session information<a class="anchor" aria-label="anchor" href="#session-information"></a>
</h2>
<pre><code><span><span class="co">## R Under development (unstable) (2025-03-08 r87910)</span></span>
<span><span class="co">## Platform: x86_64-pc-linux-gnu</span></span>
<span><span class="co">## Running under: Ubuntu 24.04.2 LTS</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Matrix products: default</span></span>
<span><span class="co">## BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3 </span></span>
<span><span class="co">## LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/libopenblasp-r0.3.26.so;  LAPACK version 3.12.0</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## locale:</span></span>
<span><span class="co">##  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              </span></span>
<span><span class="co">##  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span></span>
<span><span class="co">##  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   </span></span>
<span><span class="co">##  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span></span>
<span><span class="co">##  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span></span>
<span><span class="co">## [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## time zone: UTC</span></span>
<span><span class="co">## tzcode source: system (glibc)</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## attached base packages:</span></span>
<span><span class="co">## [1] stats4    stats     graphics  grDevices utils     datasets  methods  </span></span>
<span><span class="co">## [8] base     </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## other attached packages:</span></span>
<span><span class="co">##  [1] MsDataHub_1.7.4             QFeatures_1.17.5           </span></span>
<span><span class="co">##  [3] MultiAssayExperiment_1.33.9 SummarizedExperiment_1.37.0</span></span>
<span><span class="co">##  [5] Biobase_2.67.0              GenomicRanges_1.59.1       </span></span>
<span><span class="co">##  [7] GenomeInfoDb_1.43.4         IRanges_2.41.3             </span></span>
<span><span class="co">##  [9] MatrixGenerics_1.19.1       matrixStats_1.5.0          </span></span>
<span><span class="co">## [11] PSMatch_1.11.0              Spectra_1.17.9             </span></span>
<span><span class="co">## [13] BiocParallel_1.41.5         S4Vectors_0.45.4           </span></span>
<span><span class="co">## [15] BiocGenerics_0.53.6         generics_0.1.3             </span></span>
<span><span class="co">## [17] BiocStyle_2.35.0           </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## loaded via a namespace (and not attached):</span></span>
<span><span class="co">##  [1] DBI_1.2.3               rlang_1.1.5             magrittr_2.0.3         </span></span>
<span><span class="co">##  [4] clue_0.3-66             compiler_4.5.0          RSQLite_2.3.9          </span></span>
<span><span class="co">##  [7] png_0.1-8               systemfonts_1.2.2       vctrs_0.6.5            </span></span>
<span><span class="co">## [10] reshape2_1.4.4          stringr_1.5.1           ProtGenerics_1.39.2    </span></span>
<span><span class="co">## [13] pkgconfig_2.0.3         MetaboCoreUtils_1.15.0  crayon_1.5.3           </span></span>
<span><span class="co">## [16] fastmap_1.2.0           dbplyr_2.5.0            XVector_0.47.2         </span></span>
<span><span class="co">## [19] rmarkdown_2.29          UCSC.utils_1.3.1        ragg_1.3.3             </span></span>
<span><span class="co">## [22] purrr_1.0.4             bit_4.6.0               xfun_0.52              </span></span>
<span><span class="co">## [25] cachem_1.1.0            jsonlite_2.0.0          blob_1.2.4             </span></span>
<span><span class="co">## [28] DelayedArray_0.33.6     parallel_4.5.0          cluster_2.1.8.1        </span></span>
<span><span class="co">## [31] R6_2.6.1                bslib_0.9.0             stringi_1.8.7          </span></span>
<span><span class="co">## [34] jquerylib_0.1.4         Rcpp_1.0.14             bookdown_0.42          </span></span>
<span><span class="co">## [37] knitr_1.50              BiocBaseUtils_1.9.0     Matrix_1.7-3           </span></span>
<span><span class="co">## [40] igraph_2.1.4            tidyselect_1.2.1        abind_1.4-8            </span></span>
<span><span class="co">## [43] yaml_2.3.10             codetools_0.2-20        curl_6.2.2             </span></span>
<span><span class="co">## [46] lattice_0.22-7          tibble_3.2.1            plyr_1.8.9             </span></span>
<span><span class="co">## [49] withr_3.0.2             KEGGREST_1.47.0         evaluate_1.0.3         </span></span>
<span><span class="co">## [52] desc_1.4.3              BiocFileCache_2.15.1    ExperimentHub_2.15.0   </span></span>
<span><span class="co">## [55] Biostrings_2.75.4       pillar_1.10.2           BiocManager_1.30.25    </span></span>
<span><span class="co">## [58] filelock_1.0.3          DT_0.33                 ncdf4_1.24             </span></span>
<span><span class="co">## [61] BiocVersion_3.21.1      glue_1.8.0              lazyeval_0.2.2         </span></span>
<span><span class="co">## [64] tools_4.5.0             AnnotationHub_3.15.0    mzR_2.41.4             </span></span>
<span><span class="co">## [67] fs_1.6.5                grid_4.5.0              tidyr_1.3.1            </span></span>
<span><span class="co">## [70] crosstalk_1.2.1         MsCoreUtils_1.19.2      AnnotationDbi_1.69.0   </span></span>
<span><span class="co">## [73] GenomeInfoDbData_1.2.14 cli_3.6.4               rappdirs_0.3.3         </span></span>
<span><span class="co">## [76] textshaping_1.0.0       S4Arrays_1.7.3          dplyr_1.1.4            </span></span>
<span><span class="co">## [79] AnnotationFilter_1.31.0 sass_0.4.9              digest_0.6.37          </span></span>
<span><span class="co">## [82] SparseArray_1.7.7       htmlwidgets_1.6.4       memoise_2.0.1          </span></span>
<span><span class="co">## [85] htmltools_0.5.8.1       pkgdown_2.1.1.9000      lifecycle_1.0.4        </span></span>
<span><span class="co">## [88] httr_1.4.7              mime_0.13               bit64_4.6.0-1          </span></span>
<span><span class="co">## [91] MASS_7.3-65</span></span></code></pre>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Laurent Gatto, Johannes Rainer.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.1.9000.</p>
</div>

      </footer>
</div>






  </body>
</html>
